DEPLOYMENT_GUIDE.md

```diff
@@ -239,7 +231,7 @@ Services are organized into profiles:
 - **download**: Download-related services (API, Worker)
 - **rabbit**: RabbitMQ message queue
 
-### Multinetwork Gateway
+### Multinetwork Gateway (deprecated)
```

Comment:
As Nazar recently pointed out to me, we are actually retiring the gateway, not deprecating it. Deprecating implies it will still exist, just not be fully supported.